src/Bento.Website/Properties/launchSettings.json

```diff
@@ -18,12 +18,12 @@
     },
     "Umbraco.Web.UI": {
       "commandName": "Project",
-      "dotnetRunMessages": true,
       "launchBrowser": true,
       "applicationUrl": "https://localhost:44360;http://localhost:40560",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
-      }      
+      },
+      "dotnetRunMessages": true
```

Comment:
Should this have been committed?

Reply:
This looks like it was just moved?  will double check the file